### DISCUSSION TITLE
Current day (or recent) Wwise support?

### DISCUSSION THREAD

**Discussion by NathanBonkerk** [Wasted-Audio/hvcc#135] [Q&A] [answered]
Hey y'all,

I've been trying to get a PD patch working as a Wwise plugin the last 2 days, and I've learned a lot about how things are supposed to work. However, it seems I've hit a bit of a wall by using Wwise version 2022.1.4.8202.2651. Up untill which version will my idea work?

And if anyone is willing to have a little chat answering some more general questions, I would love that as well. Maybe through Discord or mail? Cheers!

**Comment by dromer** [comment, marked as answer]
This has already been reported here: https://github.com/Wasted-Audio/hvcc/issues/123

Please try this upcoming rewrite that brings c2wwise to current wwise SDK: https://github.com/Wasted-Audio/hvcc/pull/130
